rclcpp/src/rclcpp/time.cpp

```diff
@@ -194,7 +194,7 @@ Duration
 Time::operator-(const rclcpp::Time & rhs) const
 {
   if (rcl_time_.clock_type != rhs.rcl_time_.clock_type) {
-    throw std::runtime_error("can't subtract times with different time sources");
+    throw std::runtime_error(std::string("can't subtract times with different time sources [") + std::to_string(rcl_time_.clock_type) + " != " + std::to_string(rhs.rcl_time_.clock_type) + "]");
```

Comment:
Are linters happy with the length of this line?

Reply:
Sorry, good point. Fixed in ef7511c3ff2db0d8e475640f06c003aa574352ed.